vagrant/_commons.sh

```diff
@@ -30,13 +27,24 @@ function install_k8s {
     if [[ ! -d $kubespray_folder ]]; then
         echo "Download kubespray binaries"
 
-        sudo yum install -y wget
-        wget https://github.com/kubernetes-sigs/kubespray/archive/$kubespray_tarball
-        sudo tar -C $kubespray_dest_folder -xzf $kubespray_tarball
+        # shellcheck disable=SC1091
+        source /etc/os-release || source /usr/lib/os-release
+        case ${ID,,} in
+            rhel|centos|fedora)
+                sudo yum install -y git
+            ;;
+            clear-linux-os)
+                sudo systemctl unmask docker.service
+            ;;
+        esac
+        #sudo git clone https://github.com/kubernetes-sigs/kubespray $kubespray_folder
+        sudo git clone https://github.com/electrocucaracha/kubespray $kubespray_folder
```

Comment:
my expectation is when running on Clear Linux we use k8s what Clear linux ships in ```cloud-native-basic``` bundle. The bundle provides easy-to-use installation scripts in ```/usr/share/clr-k8s-examples```

Reply:
AFAIK this PR does that https://github.com/kubernetes-sigs/kubespray/pull/3855/files

Reply:
Does it work for you? A quick look tells me it installs ```containers-basic``` (and not ```cloud-native-basic```)

Reply:
I did some checking and apparently kubespray always uses the upstream binaries and not what the OS provides (for cri-o, kubeadm/kubelet). It also does not give you kata-containers.

Reply:
We can do those changes after Kubernetes is deployed. Eventually we can propose those changes to the Kubespray community

Reply:
> We can do those changes after Kubernetes is deployed. 

I don't follow what changes? My proposal is for Clear install we use ```cloud-native-basic``` and the setup scripts it provides.